docs/README.md

```diff
@@ -0,0 +1,624 @@
+#     "volume":12864.75939
+#   }
+# ]
+
+# By default, calculations are "Per Merchant", but another normalization can be selected. Let's try some different metrics with "Per Transaction"
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01&normalization=transaction"
+# [
+#   {
+#     "volume": 73.43840
+#   }
+# ]
+
+# In addition to filtering by a specific month, a range can be provided. Once a date range has been specified, it can be "grouped" so the result set contains the average for each month, instead of the average across the months:
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=2018-01,2018-03&group_by=date"
+# [
+#   {
+#     "date": "2018-01-01",
+#     "volume": 28352.73849
+#   },
+#   {
+#     "date": "2018-02-01",
+#     "volume": 27424.03418
+#   },
+#   {
+#     "date": "2018-03-01",
+#     "volume": 38738.38481
+#   }
+# ]
+
+# Now, let's see the volume for credit and sig debit:
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01;card=credit,sig_debit&group_by=card"
+# [
+#  {
+#    "card": "credit",
+#    "volume": 28327.3061
+#  },
+#  {
+#    "card": "sig_debit",
+#    "volume": 1924.48568
+#  }
+#]
+
+# How about focused on Omaha, Iowa, Kansas, and Missouri?
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01;card=credit,sig_debit;state=MO,KS,NE,IA&group_by=card"
+# [
+#   {
+#     "card": "credit",
+#     "volume": 23394.83581
+#   },
+#   {
+#     "card": "sig_debit",
+#     "volume": 1483.83589
+#   }
+# ]
+
+# Putting these examples together, one can see metrics per transaction in specific states grouped by card and date.
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01,2018-03;card=credit,sig_debit;state=MO,KS,NE,IA&group_by=date,card&normalization=transaction"
+# [
+#   {
+#     "card": "credit",
+#     "date": "2018-01-01",
+#     "volume": 53.77786
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-01-01",
+#     "volume": 8.38631
+#   },
+#   {
+#     "card": "credit",
+#     "date": "2018-02-01",
+#     "volume": 68.85441
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-02-01",
+#     "volume": 2.41721
+#   },
+#   {
+#     "card": "credit",
+#     "date": "2018-03-01",
+#     "volume": 58.09631
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-03-01",
+#     "volume": 5.42187
+#   }
+# ]
+```
+
+### Aggregation
+
+Abstract aggregation operation.
+    
+
+<details markdown='1'><summary>Aggregations</summary>
+
+#### None
+
+
+
+#### 3 Month Moving Average
+
+Periods = 3, Frequency = Month
+    
+
+#### 6 Month Moving Average
+
+Periods = 6, Frequency = Month
+    
+
+#### 12 Month Moving Average
+
+Periods = 12, Frequency = Month
+    
+
+#### 18 Month Moving Average
+
+Periods = 18, Frequency = Month
+    
```

Comment:
Al:
> Have a space between None and 3 months for easier visual. Also maybe put an example under the
“Periods” lines

Reply:
Once we add a description here, this will be fixed.

"None" should really just be called "Monthly", "No Window", or similar with an explanation of that.